palace/models/spaceoperator.cpp

```diff
@@ -422,9 +400,19 @@ SpaceOperator::GetComplexExtraSystemMatrix(double omega,
     ai = BuildOperator(GetNDSpace(), (SumCoefficient *)nullptr, (SumCoefficient *)nullptr,
                        &dfbi, &fbi, pa_order_threshold, skip_zeros);
   }
-  auto A = std::make_unique<ComplexParOperator>(std::move(ar), std::move(ai), GetNDSpace());
-  A->SetEssentialTrueDofs(nd_dbc_tdof_lists.back(), diag_policy);
-  return A;
+  if constexpr (std::is_same<OperType, ComplexOperator>::value)
```

Comment:
`complex` declared above, can reuse

Reply:
Fixed by rearranging, in [1df5506](https://github.com/awslabs/palace/pull/87/commits/1df550654489e49c55948409f16f90aeaad921d9).